[2023-05-26 05:57:16,125] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-01-24T00:00:00+00:00 [queued]>
[2023-05-26 05:57:16,138] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-01-24T00:00:00+00:00 [queued]>
[2023-05-26 05:57:16,140] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-05-26 05:57:16,141] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-05-26 05:57:16,142] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-05-26 05:57:16,166] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): db_connect> on 2023-01-24 00:00:00+00:00
[2023-05-26 05:57:16,173] {standard_task_runner.py:52} INFO - Started process 358 to run task
[2023-05-26 05:57:16,175] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tercera_entrega_ecentoz', 'db_connect', 'scheduled__2023-01-24T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dag_tercera_entrega.py', '--cfg-path', '/tmp/tmpbc1hovey', '--error-file', '/tmp/tmplv2wgnpk']
[2023-05-26 05:57:16,177] {standard_task_runner.py:80} INFO - Job 25: Subtask db_connect
[2023-05-26 05:57:16,248] {task_command.py:371} INFO - Running <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-01-24T00:00:00+00:00 [running]> on host f4aa0d8de4f3
[2023-05-26 05:57:16,322] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ecentoz
AIRFLOW_CTX_DAG_ID=tercera_entrega_ecentoz
AIRFLOW_CTX_TASK_ID=db_connect
AIRFLOW_CTX_EXECUTION_DATE=2023-01-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-24T00:00:00+00:00
[2023-05-26 05:57:40,666] {python.py:173} INFO - Done. Returned value was: None
[2023-05-26 05:57:40,681] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tercera_entrega_ecentoz, task_id=db_connect, execution_date=20230124T000000, start_date=20230526T055716, end_date=20230526T055740
[2023-05-26 05:57:40,744] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-26 05:57:40,779] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
